### PR TITLE
Use variables for zindex, make action buttons go above meta button

### PR DESCRIPTION
**Pull request by k88hudson**
No bug, but I noticed that it's impossible to add a new element if a link element is currently selected on the page editor

### CODE REVIEW

File: www_src/pages/element/element.less [hunk 1]
```diff
@@ -2,65 +2,6 @@
   color: @slate;
   height: 100%;
   overflow-y: scroll;
-
```
Comment (k88hudson) [inline www_src/pages/element/element.less]: unused duplicated code

File: www_src/variables.less [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Action/+ menu
+@ZINDEX_ACTION_OVERLAY: 999;
+
+// Set link destination button
+@ZINDEX_META_BUTTON: @ZINDEX_ACTION_OVERLAY - 2;
+
+// Loading indicator
+@ZINDEX_LOADING: 10000;
+
+@ZINDEX_MAX: 2147483647; /* signed 32bit maxint */
```
Comment (gvn) [inline www_src/variables.less]: Should really be `@ZINDEX_POMAX` :stuck_out_tongue_winking_eye:

**Comment by gvn** [comment]
Seems good from my general smoke testing. Since this is a broad change and we have no integration tests I'd suggest getting a 2nd reviewer! :+1:

**Comment by k88hudson** [comment]
@gvn thanks!

@Pomax mind giving this a quick check-over?

**Pull request merged**